R/renderers.R

```diff
@@ -115,9 +115,9 @@ ffmpeg_renderer <- function(format = 'mp4', ffmpeg = NULL, options = list(pix_fm
   function(frames, fps) {
     output_file <- tempfile(fileext = paste0('.', sub('^\\.', '', format)))
     frame_loc <- dirname(frames[1])
-    file_glob <- sub('^.*(\\..+$)', '%*\\1', basename(frames[1]))
+    file_glob <- sub('^.*(\\..+$)', 'gganim_plot%4d\\1', basename(frames[1]))
     file_glob <- file.path(frame_loc, file_glob)
-    system2(ffmpeg, c(
+    system2(ffmpeg, c("-pattern_type sequence",
       paste0('-i ', file_glob),
       '-y',
       '-loglevel quiet',
```

Comment:
You are suppressing output from `system2()` call here. Would you consider adding a check that the non-null size file has been created? Something along the lines of:
```r
If(!file.exists(output_file))
   stop("Something went wrong with ffmpeg attempt to create video file. Please check parameters and try again.", call. = FALSE )
``` 
You may also want to add `-loglevel quiet` conditional upon value of `progress` variable. Is there a way to set progress other than through `setOption(knitr.in.progress)`?